app/views/users/show/_timeline.html.erb

```diff
@@ -3,30 +3,14 @@
     <h3 class="title is-3">Your contributions</h3><br/>
     <table class="is-narrow is-fullwidth">
       <tbody>
-        <% scoring_pull_requests.map do |pr| %>
-          <%= render partial: 'users/show/pr/pr', locals: { pr: pr } %>
-        <% end %>
-
-        <% if show_timer %>
```

Comment:
It looks like we're no longer using `show_timer` -- assuming we're not, we can remove the helper method definition